features/show_bookings.feature

```diff
@@ -16,4 +16,10 @@ Feature: User can see a list of their laundry bookings
   
   Scenario: User don't have any bookings
     Given user visits the booking page
-    Then I should see the text "You don't have any bookings yet."
\ No newline at end of file
+    Then I should see the text "You don't have any bookings yet."
+
+  Scenario: User should not see passed bookings
+    Given user visits the booking page
+    And I click on calendar link "01 Sep 08:00"
+    Then time is frozen at 2017-09-02
```

Comment:
You need to reload the page by visiting it again.

Reply:
Change `Then` -> `And`

Reply:
THANK YOU! 😭